controllers/chat/ChatController.js

```diff
@@ -1,3 +1,4 @@
+const moment = require('moment');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `moment` library is a good choice for handling date and time manipulations. However, it's important to note that as of September 2020, the maintainers of Moment.js have stated that they will not be adding new features or capabilities. They recommend using other libraries for new projects. If this project is expected to be maintained long-term, consider using a different library like `date-fns` or `luxon`.
```diff
- const moment = require('moment');
+ const { parseISO, formatISO } = require('date-fns');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->